rmgpy/data/thermo.py

```diff
@@ -1646,7 +1665,7 @@ def get_thermo_data_from_libraries(self, species, training_set=None):
 
         return None
 
-    def get_all_thermo_data(self, species):
+    def get_all_thermo_data(self, species, metal=None):
```

Comment:
this `metal` seems to be ignored? Why is it here?

Reply:
..and maybe it *should* be here but not ignored?

Reply:
I don't think it should be here. I think I added in metal everywhere I thought I needed it and forgot to take out when it wasn't used.